### PR TITLE
Added fingerprints/support for new version of multi/moisture/motion sensors

### PR DESCRIPTION
**Pull request by tpmanley**
@workingmonk please review

### CODE REVIEW

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy [hunk 1]
```diff
@@ -27,7 +27,7 @@ metadata {
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite",  model: "3315-S", deviceJoinName: "Water Leak Sensor"
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite",  model: "3315"
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite",  model: "3315-Seu", deviceJoinName: "Water Leak Sensor"
-
+		fingerprint inClusters: "0000,0001,0003,000F,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "SmartThings", model: "moisturev4"
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy]: you might need the deviceJoinName Water Leak Sensor

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy [hunk 2]
```diff
@@ -29,6 +29,7 @@ metadata {
         fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3305"
         fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3325"
         fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3326"
+        fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "SmartThings", model: "motionv4"
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: same thing here, deviceJoinName "Motion Sensor"

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 3]
```diff
@@ -30,6 +30,7 @@
  		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "CentraLite", model: "3320"
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "CentraLite", model: "3321"
         fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "CentraLite", model: "3321-S", deviceJoinName: "Multipurpose Sensor"
+        fingerprint inClusters: "0000,0001,0003,000F,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "SmartThings", model: "multiv4"
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy]: deviceJoinName: "Multipurpose Sensor"

**Comment by tpmanley** [comment]
@workingmonk the deviceJoinNames are updated now

**Comment by workingmonk** [comment]
looks good :+1:

**Pull request merged**